benchmarks/transport_montecarlo_numba_formal_integral_p.py

```diff
@@ -102,75 +125,69 @@ def time_populate_z_photosphere(self, p, test_data):
             self.formal_integral_geometry(test_data), None, None
         )
         func = formal_integral.populate_z
```

Comment:
Similar to what Andrew said, why reassign?

Reply:
updated